<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Automatic Attendance system using Face Recognition</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/zziid/OneDrive/Pictures/Camera Roll/Bangladesh_Army_University_of_Science_and_Technology_(logo).jpg</normaloff>C:/Users/zziid/OneDrive/Pictures/Camera Roll/Bangladesh_Army_University_of_Science_and_Technology_(logo).jpg</iconset>
  </property>
  <widget class="QLabel" name="filelabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>312</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string>Automatic Attendance App</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>471</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string notr="true">Select Batch</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>471</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string notr="true">Select Course</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>471</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed Light</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 255);</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>471</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string notr="true">Select Department</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>471</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed Light</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 255);</string>
   </property>
   <property name="text">
    <string>Train</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>471</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed Light</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 255);</string>
   </property>
   <property name="text">
    <string>Attendance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>471</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed Light</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 255);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
